### DISCUSSION TITLE
How exactly...?

### DISCUSSION THREAD

**Discussion by OccupiedToast** [ensemblesaw/ensembles-app#31] [Q&A]
Sorry for being such a ditz, but how does ensembles work? To specify, how do you record and automate tracks? Thanks!

**Comment by SubhadeepJasu** [comment]
Ensembles doesn't yet record or automate tracks. This feature is planned for the final release but not yet implemented. Sorry.

**Reply by OccupiedToast** [reply]
Thank you for your prompt response, no problem at all, I just thought I was being silly and not understanding how to use it :) 
Thanks again!
